src/App.tsx

```diff
@@ -58,11 +58,11 @@ const App = () => {
 
   const wallets = useMemo(
     () => [
-      getPhantomWallet(),
-      getSolflareWallet(),
-      getSlopeWallet(),
-      getSolletWallet({ network }),
-      getSolletExtensionWallet({ network }),
+      new PhantomWalletAdapter({ network }),
+      new SolflareWalletAdapter({ network }),
+      new SlopeWalletAdapter({ network }),
+      new SolletWalletAdapter({ network }),
+      new SolletExtensionWalletAdapter({ network }),
```

Comment:
Memo is missing `network` dependency.

Reply:
That's because `network` is outside of the render function (ie. is constant).